eng/common/pipelines/templates/steps/verify-resource-ref.yml

```diff
@@ -0,0 +1,32 @@
+
+steps:
+  - pwsh: |
+      Install-Module -Name powershell-yaml -RequiredVersion 0.4.1 -Force -Scope CurrentUser
+      $remoteHEAD = git rev-parse origin/master
```

Comment:
This will run as parts of PRs right? Do we need to configure the $remoteHead as a parameter? If so, we can default it to `$(System.PullRequest.TargetBranch)` I think.

Reply:
From my testing and observation, the pipeline configures the the remote branches in the checkout phase so I think that should not be necessary.

Reply:
I suspect we probably don't want to run this on all PR's but instead do it as part of the "aggregate reports" pipelines that check the entire repo.